app/utils/PagerDutyHelper.scala

```diff
@@ -31,6 +31,7 @@ object PagerDutyHelper extends Logging {
     val FAILED_TO_FIND_DATA: PagerDutyKeys.Value = Value
     val FAILED_TO_UPDATE_DATA: PagerDutyKeys.Value = Value
     val FAILED_TO_CREATE_DATA: PagerDutyKeys.Value = Value
+    val GONE_FROM_API: PagerDutyKeys.Value = Value
```

Comment:
I think this was not in the spec so we did not use it and can be removed.

Reply:
yes retested it this was no longer needed as we corrected the returned error codes. Done